### PR TITLE
DJ-patch 10/03/2020 11.17a

### PR DESCRIPTION
**Pull request by dgiri12**
I didn't want to litter github with many branches, so I kept all my branches local, and pushed only the final version to this already existing branch. GUI.java has been created, and Main.java has been cleaned up, and it's focus has been restored.

### CODE REVIEW

**Comment by dgiri12** [comment]
A few changes were done to CRUDDB.java, but Noel is the original author of that, so github said the files were conflicting. I'll send Noel the code, and then he can create a pull request that conveys changes to only CRUDDB.java and CRUDDB.class

**Review by sbowma15 — APPROVED** [review]
Excellent work with the textfields. We can figure out the supplier ID issue today probably.

**Review by sbowma15 — APPROVED** [review]
Interesting. Like what you did with line 143

**Review by sbowma15 — APPROVED** [review]

**Review by sbowma15 — APPROVED** [review]
Looks great. Nice job with condensing the methods!

**Review by sbowma15 — DISMISSED** [review]

**Review by sbowma15 — APPROVED** [review]

**Review by sbowma15 — APPROVED** [review]

**Review by sbowma15 — APPROVED** [review]

**Review by sbowma15 — APPROVED** [review]
Not sure why previous reviews not showing on my end. Trying to submit reviews again

**Review by sbowma15 — APPROVED** [review]

**Review by sbowma15 — DISMISSED** [review]

**Review by sbowma15 — APPROVED** [review]
Changes are good, not sure why it still wont go through. Think maybe two other people have to review it also

File: GUI.java [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    JPanel east = new JPanel();
+    JPanel west = new JPanel();
+    JPanel north = new JPanel();
+    JPanel south = new JPanel();
+    JPanel center = new JPanel();
+    JPanel bottom = new JPanel();
+
+    JFrame frame = new JFrame();
+
+    public Screen() { //does this constructor have to be public??
```
Comment (Lfickling) [inline GUI.java:64]: i would think not? nothing would be calling the screen outside of this class correct?

**Pull request merged**